container/build.sh

```diff
@@ -303,7 +304,7 @@ if [[ $FRAMEWORK == "VLLM" ]]; then
         fi
     fi
 
-    cd "$NIXL_DIR"
+    cd "$NIXL_DIR" || exit
```

Comment:
I believe this will cause a silent failure, is it needed?

Reply:
Yeah.. it might make sense to log a proper error message here. The cd will likely never fail, that's why I did not include a better error message